src/wmtk/TriMeshOperationExecutor.cpp

```diff
@@ -52,14 +52,15 @@ TriMesh::TriMeshOperationExecutor::get_incident_face_data(Tuple t)
 // constructor
 TriMesh::TriMeshOperationExecutor::TriMeshOperationExecutor(
     TriMesh& m,
-    const Tuple& operating_tuple)
+    const Tuple& operating_tuple,
+    Accessor<long>& hash_acc)
```

Comment:
maybe keep a constructor that creates the hash accessor as well? this will mess with leyi's multimesh stuff a lot

Reply:
I wanted to do that but it's not possible because the accessor is stored by reference. Or do I miss something obvious how it can be done?